fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -190,7 +193,13 @@ def parameters
         if default_value.nil?
           "#{param}: #{type}"
         else
-          "#{param}: #{type} = #{default_value}"
+          if type == "((String) -> Void)?"
+            "#{param}: #{type} = nil"
+          elsif optional
+            "#{param}: ConfigItem<#{type}> = .fastlaneDefault(#{default_value})"
+          else
+            "#{param}: #{type} = #{default_value}"
```

Comment:
Why some params are treated as ConfigItem others are not? Wouldn't it be natural that we always use ConfigItem for every single param for actions?

Reply:
`ConfigItem` is just designed for optional Ruby values